el-patch.el

```diff
@@ -933,9 +990,9 @@ See `el-patch-validate'."
               (let ((type-hash (gethash type patch-hash)))
                 (dolist (variant (hash-table-keys type-hash))
                   (setq patch-count (1+ patch-count))
-                  (let ((el-patch-variant variant))
-                    (unless (el-patch-validate name type 'nomsg)
-                      (setq warning-count (1+ warning-count)))))))))
+                  (unless (el-patch-validate name type 'nomsg nil
+                                             variant)
+                    (setq warning-count (1+ warning-count))))))))
```

Comment:
I might be mistaken, but I think this was a bug. Setting `el-patch-variant` does not change the `el-patch-validate`. Instead the variant should be passed.